extension/src/state_aggregate.rs

```diff
@@ -185,6 +189,18 @@ pub fn duration_in(state: String, aggregate: Option<StateAgg>) -> i64 {
         .unwrap_or(0)
 }
 
+#[pg_extern(immutable, parallel_safe, schema = "toolkit_experimental")]
+pub fn values(
+    agg: StateAgg<'_>,
+) -> impl std::iter::Iterator<Item = (name!(state, String), name!(duration, i64))> + '_ {
```

Comment:
It would be awesome if we could make both this and `DurationInState` return a postgres `Interval` type instead of microseconds, but that's a bit of a heavier lift.

Reply:
Yeah, that's probably easier than the `AnyElement` task